tests/behaviour/steps/params.rs

```diff
@@ -510,11 +510,8 @@ impl Value {
                 } else {
                     self.raw_value.as_str()
                 };
-                let (integer, fractional) = if let Some(split) = trimmed_value.split_once(".") {
-                    split
-                } else {
-                    (trimmed_value, "0")
-                };
+                let (integer, fractional) =
+                    if let Some(split) = trimmed_value.split_once(".") { split } else { (trimmed_value, "0") };
```

Comment:
Would be cleaner with `.unwrap_or` BTW.
```suggestion
                let (integer, fractional) = trimmed_value.split_once(".").unwrap_or((trimmed_value, "0"));
```

Reply:
Haha, I literally suggested the same thing in another Krishnan's pr. Accepted